packages/vc-handler-ld-local/src/suites/BbsBlsSignature2020.ts

```diff
@@ -0,0 +1,117 @@
+import { BbsBlsSignature2020 as MattrBbsBlsSignature2020, Bls12381G2KeyPair } from '@mattrglobal/jsonld-signatures-bbs'
+import { IAgentContext, IKey, TKeyType, VerifiableCredential } from '@veramo/core'
+import { asArray } from '@veramo/utils'
+import suiteContext2020 from 'ed25519-signature-2020-context'
+
+import { RequiredAgentMethods, SphereonLdSignature } from '../ld-suites'
+import { hexToMultibase, MultibaseFormat } from '@sphereon/ssi-sdk-core'
+import {KeyType} from "@sphereon/ssi-sdk-bls-key-manager";
+
+export enum VerificationType {
+  Bls12381G2Key2020 = 'Bls12381G2Key2020',
+}
+
+export class SphereonBbsBlsSignature2020 extends SphereonLdSignature {
+  constructor() {
+    super()
+    suiteContext2020?.constants
+  }
+
+  getSupportedVerificationType(): string {
+    return VerificationType.Bls12381G2Key2020
+  }
+
+  getSupportedVeramoKeyType(): TKeyType {
+    return KeyType.Bls12381G2
+  }
+
+  getContext(): string {
+    return 'https://w3id.org/security/bbs/v1'
+  }
+
+  getSuiteForSigning(key: IKey, issuerDid: string, verificationMethodId: string, context: IAgentContext<RequiredAgentMethods>): any {
+    const controller = issuerDid
+
+    const id = verificationMethodId
+
+    if (!key.privateKeyHex) {
+      throw new Error('Private key must be defined')
+    }
+
+    const options = {
```

Comment:
Please give this a more informative name. The name options typically is an external method argument which makes this pretty confusing when you haven't started reading the code at the top

Reply:
Fixed!